app/src/recoil/consos.js

```diff
@@ -168,6 +171,16 @@ export const derivedDataFromDrinksState = selector({
         dailyDoses[day] = 0;
       }
       dailyDoses[day] = dailyDoses[day] + dose;
+      // abstinence days
+      if (!abstinenceSerieBroken && doneDay != day) {
+        if (dose === 0 && (doneDay == null || doneDay == dayjs(day).add(1, 'day').format('YYYY-MM-DD'))) {
```

Comment:
```suggestion
        if (dose === 0 && (!doneDay || doneDay === dayjs(day).add(1, 'day').format('YYYY-MM-DD'))) {
```

jamais `==`, toujours `===`
jamais `!=`, toujours `!==`